Public/Src/Utilities/Utilities/Tracing/Events.cs

```diff
@@ -294,13 +294,15 @@ public static class Tasks
 
             public const EventTask ExecutionAnalyzers = (EventTask)22;
 
+            public const EventTask SandboxedProcessExecutor = (EventTask)23;

```

Comment:
>SandboxedProcessExecutor [](start = 35, length = 24)

Nit - you could use slot 5 to compact our list

Reply:
I thought it was reserved for something else, but if not, I'll use it.

---
In reply to: [276482913](https://github.com/Microsoft/BuildXL/pull/159#discussion_r276482913) [](ancestors = 276482913)